src/main/java/rpgcharacters/commands/CommandCreateUser.java

```diff
@@ -30,6 +30,19 @@ public void run(Connection conn) {
 
             Statement stmt = conn.createStatement();
             stmt.execute(query);
+
+            query = "GRANT dbUser "
+                  + "TO (SELECT username "
+                  +     "FROM user "
+                  +     "WHERE username = '" + this.username + "');";
+            stmt.executeQuery(query);
+
+            query = "GRANT dbAdmin "
+                  + "TO (SELECT admin "
+                  +     "FROM user "
+                  +     "WHERE admin = TRUE);";
```

Comment:
this query should not be run every time on user creation, either... admin role is granted to the first user in the db, and then only to those users that user grants admin privileges to. replace this with a `SELECT COUNT(*) AS userCount ...`, and then if `userCount == 1` AFTER the user has been created, grant dbAdmin to that user